ArchiSteamFarm/www/pages/bots.html

```diff
@@ -206,21 +209,25 @@ <h1>
 
                                 gamesNumber += GamesToFarm.length;
 
-                                farmingIsActive = true; //at least one bot is farming
+                                //at least one bot is farming
+                                farmingIsActive = true;
 
                                 boxColorHTML = "box-success";
 
                                 expandBoxHTML = '<button title="Show more infos" type="button" class="btn btn-box-tool" data-widget="collapse" id="collapse-trigger"><i class="fa fa-plus"></i></button>'
+                                
+                                for (var appID in allGameAppID) {
+                                    var value = allGameAppID[appID];
+                                    allGamesHTML += '<img src="http://cdn.edgecast.steamstatic.com/steam/apps/' + value + '/header.jpg" class="img-responsive">';
+                                }
 
                                 boxBodyHTML = '<div class="box-body" id="' + BotName + '">'
```

Comment:
You must add `display: none` to the `box-body` style, otherwise you will not get that expanding animation after you click the `+` button for the first time.

Reply:
I think thats maybe a bug in AdminLTE, since `box-body` already has that style.

![unbenannt](https://user-images.githubusercontent.com/31552675/35828235-1306ec32-0abf-11e8-9ef0-30330c141f9d.PNG)

But you are right about the fact that it wont animate the first time you expand the box.

Reply:
I too think it's a bug on AdminLTE, but adding `display: none` directly to the `style` attribute fixes it.